lib/rpc/endpoints/simulate_transaction.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Soroban.RPC.SimulateTransaction do
+  @moduledoc """
+  To perform a request to SimulateTransaction endpoint for Soroban.
+  """
+  @behaviour Soroban.RPC.Endpoint.Spec
+
+  alias Soroban.RPC.{Request, SimulateTransactionResponse}
+
+  @type transaction :: String.t()
+  @type response :: tuple()
+
+  @impl true
+  def request(transaction) do
+    "simulateTransaction"
```

Comment:
Please let's place this into a module attribute `@endpoint "simulateTransaction"`